kousa/lib/beef/access/users.ex

```diff
@@ -64,6 +64,7 @@ defmodule Beef.Access.Users do
 
   def get_users_in_current_room(user_id) do
     case tuple_get_current_room_id(user_id) do
+      {:ok, nil} -> {nil, []}
```

Comment:
here was the problem; if it came back with `{:ok, nil}`, it would pass nil into the succeeding sql query; querying with x == nil is no bueno.